### PR TITLE
Ballerina siddhi

### PR DESCRIPTION
**Pull request by RameshaKaru**

### CODE REVIEW

File: components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/ThrottleStreamProcessor.java [hunk 1]
```diff
@@ -57,7 +57,7 @@
 /**
  * Throttle stream processor class
  */
-@Extension(name = "timeBatch", namespace = "gateway/throttler", description =
+@Extension(name = "timeBatch", namespace = "throttler", description =
```
Comment (Rajith90) [inline components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/ThrottleStreamProcessor.java]: Lets use the old namespace

File: components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/EmitOnStateChange.java [hunk 2]
```diff
@@ -66,7 +66,7 @@
  * insert into AlertStream;
  */
 
-@Extension(name = "emitOnStateChange", namespace = "gateway/throttler", description = "The logger stream processor logs the "
+@Extension(name = "emitOnStateChange", namespace = "throttler", description = "The logger stream processor logs the "
```
Comment (Rajith90) [inline components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/EmitOnStateChange.java]: Lets use the old namespace

File: components/micro-gateway-cli/src/main/resources/templates/policy.mustache [hunk 3]
```diff
@@ -45,7 +45,7 @@ forever {
     }
 
     from s{{name}}resultStream
-    window gateway:emitOnStateChange(s{{name}}resultStream.throttleKey, s{{name}}resultStream.isThrottled, "s{{name}}resultStream")
+    throttler:emitOnStateChange(s{{name}}resultStream.throttleKey, s{{name}}resultStream.isThrottled)
```
Comment (Rajith90) [inline components/micro-gateway-cli/src/main/resources/templates/policy.mustache]: Lets use the old namespace

File: components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/ThrottleStreamProcessor.java (hunk 1 above)
Comment (RameshaKaru) [reply components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/ThrottleStreamProcessor.java]: due to an issue in new ballerina this namespace cannot be used

File: components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/EmitOnStateChange.java (hunk 2 above)
Comment (RameshaKaru) [reply components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/EmitOnStateChange.java]: due to an issue in new ballerina this namespace cannot be used

File: components/micro-gateway-cli/src/main/resources/templates/policy.mustache (hunk 3 above)
Comment (RameshaKaru) [reply components/micro-gateway-cli/src/main/resources/templates/policy.mustache]: due to an issue in new ballerina this namespace cannot be used

**Pull request merged**